remapping_commands.sh

```diff
@@ -94,14 +108,16 @@ echo "#### Mapping results: ####"
 
 # Bowtie2 arguments:
 # -k 1 means that bowtie2 will only report the best alignment for each variant (default behaviour)
+# -k 2 means that bowtie2 will report the best alignment and the second best
```

Comment:
[From the docs](http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#k-mode-search-for-one-or-more-alignments-report-each) it seems that `-k 1` is the default mode whether it's specified or not, [which only reports 1 alignment](http://bowtie-bio.sourceforge.net/manual.shtml#example-4-default--k-1). So I think although it may report an XS, it's always the same as the AS because it doesn't actually look for a 2nd alignment.

Reply:
OK, let's just leave it at `-k 2`, it shouldn't do any harm anyway